models/oooRequests.ts

```diff
@@ -38,4 +38,39 @@ export const createOooRequest = async (body: OooStatusRequestBody) => {
     logger.error(ERROR_WHILE_CREATING_OOO_REQUEST, error);
     throw error;
   }
-};
\ No newline at end of file
+};
+
+export const updateOooRequest = async (id: string, body: OooRequestUpdateBody, lastModifiedBy:string) => {
+  try {
+    const requestExists = await oooRequestModel.doc(id).get();
```

Comment:
NIT: this variable name can be better, this is request object/document from firestore.

Reply:
Done